phpunit.xml

```diff
@@ -1,22 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/6.3/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
          bootstrap="vendor/autoload.php"
          forceCoversAnnotation="true"
          beStrictAboutCoversAnnotation="true"
          beStrictAboutOutputDuringTests="true"
          beStrictAboutTodoAnnotatedTests="true"
-         convertErrorsToExceptions="true"
-         convertNoticesToExceptions="true"
-         convertWarningsToExceptions="true"
+         convertDeprecationsToExceptions="true"
          verbose="true"
          colors="true"
          >
-    <filter>
-      <whitelist processUncoveredFilesFromWhitelist="true">
+    <coverage>
```

Comment:
This changed in `phpunit/phpunit:9.5.0` - it was highlighted by pointing at a local XSD definition